src/Test/Utilities/Portable/TargetFrameworkUtil.cs

```diff
@@ -17,6 +17,8 @@ public enum TargetFramework
         Empty,
 
         NetStandard20,
+        NetStandard30,
+        NetStandardLatest = NetStandard30,
```

Comment:
Is the intent to keep `NetStandardLatest` forever or delete once we switch the default from `NetStandard20` to `NetStandard30`?